<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Enter_Stu</class>
 <widget class="QWidget" name="Enter_Stu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>学生选修课程系统 SECS - 学生端</string>
  </property>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>11</y>
     <width>371</width>
     <height>391</height>
    </rect>
   </property>
   <property name="title">
    <string>我的选修课程</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="3">
     <widget class="QTableWidget" name="tableWidget_SelectedCourse">
      <column>
       <property name="text">
        <string>课程名称</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>当前选中的课程：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="lable_CurrentCourse2">
      <property name="text">
       <string>未选中...</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="btn_DelMy">
      <property name="text">
       <string>退选当前课程</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>11</y>
     <width>591</width>
     <height>391</height>
    </rect>
   </property>
   <property name="title">
    <string>选修课程库</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>搜索：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="comboBox_SearchType">
      <item>
       <property name="text">
        <string>课程名称</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>课程代码</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLineEdit" name="lineEdit_Search"/>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>当前选中的课程：</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="lable_CurrentCourse1">
      <property name="text">
       <string>未选中...</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="btn_AddMy">
      <property name="text">
       <string>添加到我选修的课程</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="5">
     <widget class="QTableView" name="tableView_Course"/>
    </item>
    <item row="0" column="3">
     <widget class="QPushButton" name="btn_Fresh">
      <property name="text">
       <string>刷新选修课程信息</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>420</y>
     <width>185</width>
     <height>17</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>欢迎您，</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_No">
      <property name="text">
       <string>00000000</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_Name">
      <property name="text">
       <string>我是人</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
